lib/scala/tree/src/main/scala/io/github/tetherlessworld/twxplore/lib/tree/models/domain/SelectionGeometry.scala

```diff
@@ -0,0 +1,6 @@
+package io.github.tetherlessworld.twxplore.lib.tree.models.domain
+
+import edu.rpi.tw.twks.uri.Uri
+import io.github.tetherlessworld.twxplore.lib.geo.models.domain.Geometry
+
+case class SelectionGeometry(geometry: Geometry, uri: Uri)
```

Comment:
final

Reply:
These aren't really domain models any more if they're only used to support the GraphQL API. If that's the case move them to models.graphql.